contracts/ResilientOracle.sol

```diff
@@ -329,8 +327,46 @@ contract ResilientOracle is PausableUpgradeable, AccessControlledV8, ResilientOr
         enabled = tokenConfigs[asset].enableFlagsForOracles[uint256(role)];
     }
 
+    /**
+     * @notice Updates the TWAP oracle price and capped oracle snapshot.
+     * @dev Cache the asset price and return if already cached
+     * @param asset asset address
+     */
+    function _updateAssetPrice(address asset) internal {
+        if (Transient.readCachedPrice(CACHE_SLOT, asset) != 0) {
+            return;
+        }
+
+        (address mainOracle, bool mainOracleEnabled) = getOracle(asset, OracleRole.MAIN);
+        if (mainOracle != address(0) && mainOracleEnabled) {
+            // if main oracle is not CorrelatedTokenOracle it will revert so we need to catch the revert
+            try ICappedOracle(mainOracle).updateSnapshot() {} catch {}
+        }
+
+        (address pivotOracle, bool pivotOracleEnabled) = getOracle(asset, OracleRole.PIVOT);
+        if (pivotOracle != address(0) && pivotOracleEnabled) {
+            //if pivot oracle is not TwapOracle it will revert so we need to catch the revert
+            try TwapInterface(pivotOracle).updateTwap(asset) {} catch {}
+        }
```

Comment:
We can remove this code, because we don't have any TWAP oracle. We can also remove the `TwapInterface` file